src/main/java/dinkplugin/SettingsManager.java

```diff
@@ -158,6 +179,114 @@ private static boolean isKillCountFilterInvalid(int varbitValue) {
+                    plugin.addChatWarning("Clipboard was empty");
+                    return null;
+                }
+            })
+            .thenAcceptAsync(this::handleImport)
+            .exceptionally(e -> {
+                plugin.addChatWarning("Failed to read clipboard");
+                return null;
+            });
+    }
+
+    @Synchronized
+    private void handleImport(Map<String, String> map) {
+        if (map == null) return;
+
+        Collection<String> oldPrimary = readDelimited(config.primaryWebhook()).collect(Collectors.toCollection(LinkedHashSet::new));
+        AtomicInteger numUpdated = new AtomicInteger();
+        map.forEach((key, value) -> {
+            String prevValue = configManager.getConfiguration(CONFIG_GROUP, key);
+            String newValue;
+
+            if ("discordWebhook".equals(key)) {
+                Collection<String> newPrimary = new LinkedHashSet<>(oldPrimary);
+                readDelimited(value).forEach(newPrimary::add);
+
+                if (newPrimary.size() > oldPrimary.size()) {
+                    newValue = String.join("\n", newPrimary);
+                } else {
+                    newValue = null;
+                }
+            } else if (key.endsWith("Webhook")) {
+                Collection<String> overrides = readDelimited(prevValue).collect(Collectors.toCollection(LinkedHashSet::new));
+                if (overrides.isEmpty()) {
+                    overrides.addAll(oldPrimary);
+                }
+
+                long added = readDelimited(value)
+                    .map(overrides::add)
+                    .filter(Boolean::booleanValue)
+                    .count();
+
+                if (added > 0) {
+                    newValue = String.join("\n", overrides);
+                } else {
+                    newValue = null;
+                }
+            } else if ("ignoredNames".equals(key)) {
+                Collection<String> ignored = readDelimited(config.ignoredNames()).collect(Collectors.toCollection(LinkedHashSet::new));
+                readDelimited(value).forEach(ignored::add);
+
+                if (ignored.size() > this.ignoredNames.size()) {
+                    newValue = String.join("\n", ignored);
+                } else {
+                    newValue = null;
+                }
+            } else {
+                if (Objects.equals(value, prevValue)) {
+                    newValue = null;
+                } else {
+                    newValue = value;
+                }
+            }
+
+            if (newValue != null) {
+                configManager.setConfiguration(CONFIG_GROUP, key, newValue);
+                numUpdated.incrementAndGet();
+            }
+        });
+
+        plugin.addChatSuccess(
+            String.format(
+                "Updated %d config settings (from %d total specified in import). " +
+                    "Please restart the client for these changes to be reflected in the config panel.",
```

Comment:
You do not need to restart the game client for this, only close and open the config panel